settings:
  recover_height: 3.0
  auto_track: true
  auto_land: false
  auto_disarm: false
  target_lost_threshold: 0.1
  min_discover_time: 2.0
  min_tracking_time: 5.0
  home_point: [0.0, 0.0]

cam0:
  camera_model: "pinhole"
  intrinsics: [554.38, 554.38, 320.0, 320.0]

T_BC0:
  rows: 4
  cols: 4
  data: [
    0.0,  -1.0,  0.0, 0.0,
    -1.0,  0.0,  0.0, 0.0,
    0.0,   0.0, -1.0, 0.0,
    0.0,   0.0,  0.0, 1.0
  ]

att_ctrl:
  roll_ctrl:
    limits: [-35.0, 35.0]
    gains: [200.0, 0.0, 10.0]

  pitch_ctrl:
    limits: [-35.0, 35.0]
    gains: [200.0, 0.5, 10.0]

  yaw_ctrl:
    gains: [200.0, 0.5, 10.0]

pos_ctrl:
  roll_ctrl:
    limits: [-35.0, 35.0]
    gains: [0.2, 0.0, 0.02]

  pitch_ctrl:
    limits: [-35.0, 35.0]
    gains: [0.2, 0.0, 0.02]

  throttle_ctrl:
    hover_throttle: 0.5
    gains: [0.5, 0.0, 0.02]

tk_ctrl:
  roll_ctrl:
    limits: [-35.0, 35.0]
    gains: [0.08, 0.0, 0.001]

  pitch_ctrl:
    limits: [-35.0, 35.0]
    gains: [0.08, 0.0, 0.001]

  throttle_ctrl:
    hover_throttle: 0.5
    gains: [0.5, 0.0, 0.02]

landing_zone:
  tag_ids: [0, 5]
  tag_sizes: [0.1, 0.857]
